versions.yaml

```
@@ -3,12 +3,12 @@
 # prefer sticking with a stable version until a specific reason to update arises.
 SCCACHE_VER: 0.7.7
 # renovate: datasource=github-releases depName=cli/cli
-GH_CLI_VER: 2.64.0
+GH_CLI_VER: 2.67.0
 # renovate: datasource=pypi depName=codecov-cli
 CODECOV_VER: 9.1.1
```

I'd like to see if we can update codecov-cli at the same time. (I don't know how Renovate picks which things to update.)
```suggestion
CODECOV_VER: 10.2.0
```

---

Sure, seems fine to me. I don't understand why we didn't get a PR for this either... there are definitely `codecov-cli` 10.x packages at https://pypi.org/project/codecov-cli/#history, the first of which was released in January.